assets/sass/components/_result-card.scss

```
@@ -0,0 +1,18 @@
+.result-card {
+  border: 1px solid govuk-colour('mid-grey');
+  margin-bottom: govuk-spacing(6);
+
+  &__header {
+    border-bottom: 1px solid govuk-colour('mid-grey');
+    background-color: govuk-colour("light-grey");
+    padding: govuk-spacing(4);
+
+    & .govuk-heading-m {
+      margin-bottom: 0;
+    }
+  }
+
+  &__body {
+    padding: govuk-spacing(4) govuk-spacing(4) 0;
```

```suggestion
    @include govuk-responsive-padding(4, "top");
    @include govuk-responsive-padding(4, "left");
    @include govuk-responsive-padding(4, "right");
    @include govuk-responsive-padding(0, "bottom");
```

Did you want to try this? More details https://frontend.design-system.service.gov.uk/sass-api-reference/#govuk-responsive-padding and theres a matching margin mixin too. It might make a padding a bit smoother when viewport changes. 

https://design-system.service.gov.uk/styles/spacing/#responsive-spacing